format/nexus.py

```diff
@@ -1291,7 +1291,8 @@ def set_frame(pg, transformation):
                         raise RuntimeError("Unknown material: %s" % material)
                     table = attenuation_coefficient.get_table(material)
                     wavelength = beam.get_wavelength()
-                    p.set_mu(table.mu_at_angstrom(wavelength) / 10.0)
+                    mu = table.mu_at_angstrom(wavelength) / 10.0
```

Comment:
suggest comment in here to say that attenuation coefficients work in `/cm` because CGS units, while DIALS works in mm?

Reply:
such a comment is 15 lines up (L1279)